### PR TITLE
Add book details, and service to get book info

### PR DESCRIPTION
**Pull request by blacksam07**
## Summary

- Add service to get book info
- Add component book detail
- Add logic to show badge

## Trello card

[13-book-detail](https://trello.com/c/W0PS0Zaa/13-book-detail)

## Screenshots

book-detail-min

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ import { UnauthGuard } from './guards/unauth.guard';
 import { UnauthModule } from './modules/unauth/unauth.module';
 import { TokenInterceptor } from './token.interceptor';
 import { TitleFilterPipe } from './pipes/title-filter.pipe';
+import { BookDetailComponent } from './screens/book-detail/book-detail.component';
```
Comment (CrisVega08) [inline src/app/app.module.ts]: I think is not part of the app component

File: src/app/screens/book-detail/book-detail.component.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<body class="detail">
+  <a class="back-arrow" [routerLink]="['/books']" >Atrás</a>
+  <div class="container">
+    <h2 class="bold-title">{{book.title}}</h2>
+    <span class="base-span subtitle">({{book.genre}})</span>
+    <div class="container-item" [ngClass]="{ badge: book.author == 'Chi Koepp' }">
```
Comment (CrisVega08) [inline src/app/screens/book-detail/book-detail.component.html]: { badge: book.author == 'Chi Koepp' } why this?

File: src/app/screens/book-detail/book-detail.component.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+@Component({
+  selector: 'app-book-detail',
+  templateUrl: './book-detail.component.html',
+  styleUrls: ['./book-detail.component.scss']
+})
+export class BookDetailComponent implements OnInit {
+  book: Book;
+
+  constructor(private route: ActivatedRoute, private bookService: BookService) {
+    this.book = {
```
Comment (CrisVega08) [inline src/app/screens/book-detail/book-detail.component.ts]: create a file constants. in this location and save this cont here

File: src/app/services/book.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+export class Book {
```
Comment (CrisVega08) [inline src/app/services/book.ts]:
```suggestion
export Interface Book {
```

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: src/app/screens/book-detail/book-detail.component.html (hunk 2 above)
Comment (blacksam07) [reply src/app/screens/book-detail/book-detail.component.html]: In this case, I choose `Chi Koepp`

Screen Shot 2020-05-05 at 17 24 38

Comment (CrisVega08) [reply src/app/screens/book-detail/book-detail.component.html]: ok so store the value in a variable

File: src/app/screens/book-detail/book-detail.component.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+@Component({
+  selector: 'app-book-detail',
+  templateUrl: './book-detail.component.html',
+  styleUrls: ['./book-detail.component.scss']
+})
+export class BookDetailComponent implements OnInit {
+  book: Book;
+
+  constructor(private route: ActivatedRoute, private bookService: BookService) {
+    this.book = BOOK;
```
Comment (betoobernal) [inline src/app/screens/book-detail/book-detail.component.ts:16]: use ngOnInit hook

**Review by betoobernal — APPROVED** [review]

File: src/app/screens/book-detail/book-detail.component.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<body class="detail">
```
Comment (CrisVega08) [inline src/app/screens/book-detail/book-detail.component.html]: body only one body for app, change body tag

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**